openiddict/Directory.Packages.props

```diff
@@ -0,0 +1,53 @@
+<Project>
+    <PropertyGroup>
+        <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
```

Comment:
This enables this `Directory.Packages.props` file to manage the package version in the projects.

There is an `ItemGroup` below for both .NET 6.0 and .NET 8.0, allowing different package versions to be installed (e.g. EF Core 6 vs EF Core 8

Reply:
Have reverted central package management in favour of duplication due to failing dotnet restore in the pipeline